spec/requests/historical_percent_chages_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+ RSpec.describe 'View Historical percent changes API', type: :request do
+  let!(:rent_charge_data) {[
+    create(:rent_charge, uprn: '1', year:2018),
```

Comment:
could you use the Faker gem to generate randomised test data in your `rent_charge` and `fixed_datum` factories, removing the need for `uprn` and `year` data to be defined here? and then maybe you could reduce this bit to something like `create_list(:rent_charge, 6)?`